ocfweb/docs/views/servers.py

```diff
@@ -108,7 +108,7 @@ def ldap_to_host(item):
     return hostname, Host(hostname, item['type'], description, ())
 
 
-@periodic(300)
+@cache()
```

Comment:
the parens aren't necessary

Reply:
Deleting the parens generates TypeError: outer() missing 1 required positional argument: 'fn'